lua/tabline/icons.lua

```diff
@@ -16,13 +16,13 @@ M.get_devicon = function(index, bufname, extension)
         icon_hl = 'TabLine' .. icon_hl
 
         local active_color = hl.get_color('TabLineIconSel', 'fg') or color
-        vim.api.nvim_set_hl(0, icon_hl .. 'Sel', { fg = active_color, bg = hl.c.active_bg })
+        vim.api.nvim_set_hl(0, icon_hl .. 'Sel', { fg = active_color, bg = hl.get_bg(true) })
```

Comment:
Just set `default = true` inside the opts table

Reply:
I tried this, but it still failed to update the background color with command because `hl.c.active_bg` is static.

Reply:
Not entirely sure what you mean, because if you set `default = true` here and then highlight the icon yourself, it should be possible. Note that the highlight group is for example `TabLineDevIconLua` and `TabLineDevIconLuaSel` for the lua devicon. So therefore you must use that as a highlight group.

If you change it to this:
`vim.api.nvim_set_hl(0, icon_hl .. 'Sel', { fg = active_color, bg = hl.c.active_bg, default = true })`
It will only highlight if the group is not already highlighted:

This means that you should be able to override it with:
`vim.api.nvim_set_hl(0, 'TabLineDevIconLuaSel', { fg = ..., bg = ... })` for example, or if you prefer the vim command: `vim.cmd('highlight! TabLineDevIconLuaSel guifg=... guibg=...')`

Reply:
Okay I get it with the function `get_bg` that you added. I understand what you mean about it. Change it to `active and 'TabLineSel' or 'TabLine' though, because `Sel` is for when it is active. I do think we should add `default = true` also though

Reply:
Sorry for not describing clearly what I wanted to say. Since the default source of the background color of the icons is either `TabLine` or `TabLineSel`, I intuitively thought the color of an icon should be determined by those highlighting groups.
However, changing their highlighting with the way you mentioned is fine to me.

Reply:
No problem. I did not understand at first, but I actually think we should keep `get_bg`, but only fix the small mistake I commented on in the previous comment. This way, the icons background will also change correctly if changing the colors through the command line

Reply:
I see. I will add back the `get_bg` function.